### ISSUE TITLE
package-requires lines were not updated with version

### ISSUE THREAD

**Issue by thkoch2001** [editorconfig/editorconfig-emacs#80]
I'm packaging version 0.7.3 of editorconfig-emacs for debian and emacs complains that it can not find editorconfig-core version 0.6.2.

This seems to be caused by these lines:

```
;; Version: 0.7.3
;; URL: https://github.com/editorconfig/editorconfig-emacs#readme
;; Package-Requires: ((editorconfig-core "0.6.2"))
```

I've no experience in creating emacs packages. But maybe it would be better to make this a multi file package? Otherwise it would be necessary to also update the Package-Requires lines for every version update.

https://www.gnu.org/software/emacs/manual/html_node/elisp/Multi_002dfile-Packages.html

**Comment by thkoch2001** [comment]
There is another change necessary in editorconfig-core.el.

But please comment on my proposal on changing editorconfig to a multifile package. Otherwise this might not be the last time this error happens.

**Comment by thkoch2001** [comment]
I changed all Package-Requires lines now to reference the correct version and I still get this warning:

```
Warning (emacs): Unable to activate package `editorconfig'.
Required package `editorconfig-core-0.7.3' is unavailable
```

We need to do things a bit different with debian packaging, but I still wonder, how this package works when installed via elpa. The debian elisp packaging tools install all editorconfig files to /usr/share/emacs24/site-lisp/elpa/editorconfig-0.7.3. But I believe emacs expects to find elisp package editorconfig-core version 0.7.3 in a directory named editorconfig-core-0.7.3.

So we should really evaluate the multifile approach, I believe.

**Comment by 10sr** [comment]
Thanks for packaging this plugin !

I see... I think the multifile approach should be much better.
Sorry I have no time to do it right now, but I'll fix in no distant future.

**Comment by 10sr** [comment]
I submitted a PR for this, and will merge after MELPA recipe is updated.

**Comment by 10sr** [comment]
@thkoch2001 I've updated dependencies and released v0.7.4 so I think you can now build one multifile package of editorconfig-emacs :bow:
